gc/base/standard/ConfigurationGenerational.cpp

```diff
@@ -63,6 +63,19 @@ MM_ConfigurationGenerational::newInstance(MM_EnvironmentBase *env)
 	return configuration;
 }
 
+void
+MM_ConfigurationGenerational::tearDown(MM_EnvironmentBase* env)
+{
+	MM_GCExtensionsBase* extensions = env->getExtensions();
+
+	/* Set pointer to scavenger in extensions to NULL
+	 * before Scavenger is teared down as part of clean up of defaultMemorySpace
+	 * in MM_Configuration::tearDown. */
+	extensions->scavenger = NULL;
```

Comment:
If we set this to NULL here how do we find it to free during defaultMemorySpace tearDown?  Is it based on the default collector?

Reply:
Ok thanks!